src/app/shared/conditional-show/conditional-show.directive.ts

```diff
@@ -100,6 +100,13 @@ export class ConditionalShowDirective implements AfterViewInit, OnDestroy {
 
   private show() {
     this.el.nativeElement.hidden = false;
+    if (this.caseField && this.caseField.field_type && this.caseField.field_type.type !== 'Collection') {
```

Comment:
we should move this whole new block of code into it's own private method, with a descriptive name to make the code more understandable. 
I guess we could call this new method `showGrayBar` ?

Reply:
I think the conditional-show.directive should not know the details of how to display grey bar. The directive should only have logic to decide if a field has to be displayed or not. 
Is it possible to move the method showGrayBar() outside of this class, for example having it on the field itself, and just call it here?

Reply:
Done.